x-pack/legacy/plugins/siem/public/containers/timeline/index.tsx

```diff
@@ -84,12 +84,13 @@ class TimelineQueryComponent extends QueryTemplate<
       sortField,
     } = this.props;
     const defaultKibanaIndex = kibana.services.uiSettings.get<string[]>(DEFAULT_INDEX_KEY);
-    const defaultIndex = isEmpty(indexPattern)
-      ? [
-          ...(['all', 'raw'].includes(eventType) ? defaultKibanaIndex : []),
-          ...(['all', 'signal'].includes(eventType) ? indexToAdd : []),
-        ]
-      : indexPattern?.title.split(',') ?? [];
+    const defaultIndex =
+      indexPattern == null || (indexPattern != null && indexPattern.title === '')
```

Comment:
🙌 Thanks for catching the `emptystring`! This should fix  traveling signals bug 🙂